cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/CmwAuthTokenInterceptor.java

```diff
@@ -0,0 +1,11 @@
+package com.cloudesire.platform.apiclient;
+
+import com.cloudesire.platform.apiclient.interceptors.HeaderInterceptor;
+
+public class CmwAuthTokenInterceptor extends HeaderInterceptor
+{
+    public CmwAuthTokenInterceptor( String token )
+    {
+        super("CMW-Auth-Token", token);
```

Comment:
we have a `Headers` class somewhere for that string

(format me)

Reply:
for response headers, this is a request header used internally only on the client class